README.md

```diff
@@ -358,6 +358,16 @@ OPTIONS
   --listr-renderer=default|silent|verbose  [default: default] Listr renderer
 ```
 
+To obtain a user access token from the indentity provider (eg, Keycloak or Red Hat Single Sign-On), using `oc` or `kubectl` with `curl` and `jq`:
+
+```
+KEYCLOAK_URL=$(kubectl get route/keycloak -n ${cheNamespace} -o jsonpath='{.spec.host}')
```

Comment:
I think we should notify, that it will work on Linux. For windows I don't think such shell obstructionist will work.

Reply:
by "notify" you mean something other than a console error message?